Src/VimWpf/RelativeLineNumbers/Line.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Diagnostics;
+
+namespace Vim.UI.Wpf.RelativeLineNumbers
+{
+    [DebuggerDisplay("{Number} {Baseline}")]
+    public sealed class Line

```

Comment:
Did you consider making this a `struct` to save on allocations? Looking through I didn't see an obvious place where it needed to be  a `class`. Given this is paired up with lines I'm concerned it could lead to excessive allocations in large files.

Reply:
Done